opentreemap/treemap/views/tree.py

```diff
@@ -118,6 +120,49 @@ def search_tree_benefits(request, instance):
     return context
 
 
+def simplified_tree_benefits(request, instance):
+    unit = 'kg/rok'
+    req = requests.get(settings.ECO_SERVICE_URL)
+    req.raise_for_status()
+    data = req.json()
+
+    descriptions = {
+        'O3': u'Redukcja ozonu (O3)',
```

Comment:
Same here regarding hardcoding polish phrases.

Reply:
Yeah, we'll not maintain translations here.